packages/webcert/src/feature/certificate/Buttons/ReplaceCertificateButton.tsx

```diff
@@ -16,6 +17,8 @@ interface Props extends FunctionDisabled {
 const ReplaceCertificateButton: React.FC<Props> = ({ name, description, enabled, functionDisabled }) => {
   const history = useHistory()
   const dispatch = useDispatch()
+  const certificateMetadata = useSelector(getCertificateMetaData)
+  const isDB = certificateMetadata?.type === 'db'
```

Comment:
We could make this variable a bit more readable with "isDodsbevis" or "isDeathCertificate".